src/Microsoft.MobileBlazorBindings.Hosting/GlobalSuppressions.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "BL0006:Do not use RenderTree types", Justification = "<Pending>")]
```

Comment:
Do you need this, considering that this diagnostic is disabled in editorconfig?

Reply:
I double-disabled it to be double-sure 😄 It's from a copy/paste, will fix!